tests/models/test_models_unet_2d.py

```diff
@@ -59,7 +59,7 @@ def prepare_init_args_and_inputs_for_common(self):
             "block_out_channels": (32, 64),
             "down_block_types": ("DownBlock2D", "AttnDownBlock2D"),
             "up_block_types": ("AttnUpBlock2D", "UpBlock2D"),
-            "attention_head_dim": None,
+            "attention_head_dim": 3,
```

Comment:
Is it possible to also test for the new argument (`num_attention_heads`) we introduced here to check for feature parity?

Reply:
Yes, I'll add some tests now for `num_attention_heads`